### PR TITLE
Feat/remove fiat/on ramp

### PR DESCRIPTION
**Pull request by 0x4Graham**
## General Changes

- Fixes XYZ bug
- Adds XYZ feature
- …

## Developer Notes

Add any notes here that may be helpful for reviewers.

---

## Author Checklist

Please ensure you, the author, have gone through this checklist to ensure there is an efficient workflow for the reviewers.

- [ ]  The base branch is set to `main`
- [ ]  The title is using [Conventional Commit](https://www.conventionalcommits.org/en/v1.0.0/) formatting
- [ ]  The Github issue has been linked to the PR in the Development section
- [ ]  The General Changes section has been filled out
- [ ]  Developer Notes have been added (optional)

**If the PR is ready for review:**

- [ ]  The PR is in `Open` state and not in `Draft` mode
- [ ]  The `Ready for Dev Review` label has been added

## Reviewer Checklist

Please ensure you, as the reviewer(s), have gone through this checklist to ensure that the code changes are ready to ship safely and to help mitigate any downstream issues that may occur.

- [ ]  End-to-end tests are passing without any errors
- [ ]  Code style generally follows existing patterns
- [ ]  Code changes do not significantly increase the application bundle size
- [ ]  If there are new 3rd-party packages, they do not introduce potential security threats
- [ ]  If there are new environment variables being added, they have been added to the `.env.example` file as well as the pertinant `.github/actions/*` files
- [ ]  There are no CI changes, or they have been approved by the DevOps and Engineering team(s)
- [ ]  Code changes have been quality checked in the ephemeral URL
- [ ]  QA verification has been completed
- [ ]  There are two or more approvals from the core team
- [ ]  Squash and merge has been checked

### CODE REVIEW

**Comment by 0x4Graham** [comment]
Link T-10873

File: src/ui-config/menu-items/index.tsx
Comment (grothem) [inline src/ui-config/menu-items/index.tsx:1]: Here I think you could do something like this:
```
if (fiatEnabled) {
   moreMenuItems.push({
   ... add Transak link object
  });
}
```

There's some cleanup we can do here too, lot's of unused items that can get deleted: `moreMenuExtraItems`, `moreMenuMobileOnlyItems` and `mobileNavigation`

File: src/modules/governance/DelegatedInfoPanel.tsx
[the remark this answers is not in the document]
Comment (0x4Graham) [reply src/modules/governance/DelegatedInfoPanel.tsx:1]: They're for something else, i'll remove

File: src/ui-config/menu-items/index.tsx
Comment (0x4Graham) [reply src/ui-config/menu-items/index.tsx:1]: I don't understand why we would do this here? 

On this file there is no logic

File: src/layouts/MoreMenu.tsx [hunk 1]
```diff
@@ -26,6 +26,7 @@ export function MoreMenu() {
   const handleClose = () => {
     setAnchorEl(null);
   };
+  const visibleMoreNavigation = moreNavigation.filter((item) => !item.isVisible);
```
[the remark this answers is not in the document]
Comment (0x4Graham) [reply src/layouts/MoreMenu.tsx:29]: The provided line of code doesn't work, would be good to understand what you want to do

File: src/modules/governance/DelegatedInfoPanel.tsx [hunk 2]
```diff
@@ -64,7 +64,7 @@ const DelegatedPower: React.FC<DelegatedPowerProps> = ({
           </Row>
         ) : (
           <>
-            {!isAaveSelfDelegated && (
+            {!isAaveSelfDelegated && aavePower !== '0' && (
```
Comment (foodaka) [inline src/modules/governance/DelegatedInfoPanel.tsx:67]: @0x4Graham can we remove the delegation stuff if its not relevant

Comment (0x4Graham) [reply src/modules/governance/DelegatedInfoPanel.tsx:67]: Done

**Review by MareskoY — APPROVED** [review]

**Comment by MareskoY** [comment]
Looks good QC passed

**Review by grothem — APPROVED** [review]

**Pull request merged**